src/config.rs

```diff
@@ -1641,6 +1668,22 @@ struct GlobalArgs {
     #[arg(long, value_name = "POLICY")]
     unknown_objects: Option<FilterPolicy>,
 
+    /// Maximum length of IPv4 prefixes included in output
+    #[arg(
+        long,
+        value_name = "LENGTH",
+        value_parser = clap::value_parser!(u8).range(..32)
```

Comment:
This will error if '32' is passed, as the range is not inclusive. Is that what we want? The `take_limited_u8` will allow 32 in the config file, so at the very least things are inconsistent.

Reply:
Uh, nice find!